crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -730,8 +733,14 @@ impl CumulocityConverter {
                                     output.stdout,
                                     MAX_PAYLOAD_LIMIT_IN_BYTES,
                                 );
-                                let successful_str =
-                                    format!("503,{op_name},\"{sanitized_stdout}\"");
+                                let successful_str = match result_format {
+                                    ResultFormat::Text => {
+                                        format!("503,{op_name},\"{sanitized_stdout}\"")
+                                    }
+                                    ResultFormat::CSV => {
+                                        format!("503,{op_name},{sanitized_stdout}")
+                                    }
+                                };
```

Comment:
Is `op_name` confirmed to not have commas?

When I talked with @rina23q, she recommended to use `sanitize_for_smartrest` function on single fields. For now it can stay that way because I think it was already broker, but we'll need to start removing this creating smartrest payload via ad-hoc `format!()` calls and use something that automatically sanitizes so we can't make more mistakes.

Reply:
> Is op_name confirmed to not have commas?

Yes it is safe to say the op_name will never have commas in the name.

> When I talked with @rina23q, she recommended to use sanitize_for_smartrest function on single fields. For now it can stay that way because I think it was already broker, but we'll need to start removing this creating smartrest payload via ad-hoc format!() calls and use something that automatically sanitizes so we can't make more mistakes.

Providing smart rest specific sanitization is actually useful to some degree (e.g. escaping `\n` characters)...so it is true that some unexpected sanitization might happen, however in reality there are not many use-cases for this (apart from the c8y_RelayArray, where the values are either OPEN or CLOSED). So I would say we can skip worrying about unnecessary sanitization for now.